hoomd/mpcd/ConfinedStreamingMethod.h

```diff
@@ -101,23 +102,28 @@ void ConfinedStreamingMethod<Geometry>::stream(unsigned int timestep)
         m_validate_geom = false;
         }
 
+    if (m_mpcd_pdata->getN() > m_bounced.getNumElements())
+        {
+        GPUArray<unsigned char> bounced(m_mpcd_pdata->getN(), m_exec_conf);
+        m_bounced.swap(bounced);
+        }
+
     if (m_prof) m_prof->push("MPCD stream");
 
     const BoxDim& box = m_mpcd_sys->getCellList()->getCoverageBox();
 
     ArrayHandle<Scalar4> h_pos(m_mpcd_pdata->getPositions(), access_location::host, access_mode::readwrite);
     ArrayHandle<Scalar4> h_vel(m_mpcd_pdata->getVelocities(), access_location::host, access_mode::readwrite);
+    ArrayHandle<unsigned char> h_bounced(m_bounced, access_location::host, access_mode::overwrite);
+    
     const Scalar mass = m_mpcd_pdata->getMass();
```

Comment:
I am adding suggestion below to reinsert all the blank lines that were deleted. You should not modify files in ways that are not relevant to your code changes
```suggestion
    const Scalar mass = m_mpcd_pdata->getMass();

```